.husky/pre-commit

```diff
@@ -1,4 +1,10 @@
+
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+npx lint-staged
+
```

Comment:
I hadnt thought to put this code here. I had it in my machine's config.  Im down. Good idea.

I checked out this branch, and I cant find the commit where you add it though.... very weird. 

I dont personally want to lint per commit, so can you remove that part if its real code and not just showing up in this diff wrongly? If you want to lint before you commit, can you do it with a local alias?

Reply:
k, the rest of this I will focus on componentization, and ignore other review stuff)

Reply:
I feel like I set this up wrong to begin with, pre-commit it shows it's not executable with this message: 
<img width="853" alt="Screen Shot 2022-05-17 at 8 24 10 AM" src="https://user-images.githubusercontent.com/26089140/168810178-54d12496-6c76-480d-88a4-74ca67359e4d.png">
 
 i'm good with moving it out but will create a separate task because it's not working atm

Reply:
lint-staged still here. can u remove?